index.html

```diff
@@ -0,0 +1,319 @@
+                    Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <div class="socialmedia">
+                    <div class="supporting-text">
+                        <p class="connect">LET’S CONNECT</p>
+                    </div>
+                </div>
+
+                <div class="socialmedia-icons">
+                    <ul>
+                        <li><a href="#"><img class="twitter" src="images/twitter.png"></a></li>
+                        <li><a href="#"><img class="linkedin" src="images/Linkedin icon.png"></a></li>
+                        <li><a href="#"><img class="m" src="images/m.png"></a></li>
+                        <li><a href="#"><img class="github" src="images/github.png"></a></li>
+                        <li><a href="#"><img class="vector" src="images/Vector.png"></a></li>
+                    </ul>
+                </div>
+
+            </div>
+        </section>
+    </header>
+
+    <section class="works">
+        <div class="card-works1">
+            <div class="snap1"></div>
+            <div class="left-block">
+                <div class="pt">
+                    <h2>Tonic</h2>
+                    <ul class="frame2">
+                        <li class="client">
+                            <h5 class="canopy">CANOPY</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="role">
+                            <h5 class="Back">Back End Dev</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="year-label">
+                            <h5 class="year">2015</h5>
+                        </li>
+                    </ul>
+                    <p class="pry-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="tags">
+                        <li class="tag1">
+                            <a href="#">
+                                <p class="html">html</p>
+                            </a>
+                        </li>
+                        <li class="tag2">
+                            <a href="#">
+                                <p class="css">css</p>
+                            </a>
+                        </li>
+                        <li class="tag3">
+                            <a href="#">
+                                <p class="js">javaScript</p>
+                            </a>
+                        </li>
+                    </ul>
+
+                </div>
+
+                <div class="action">
+                    <button class="see-project">
+                        <p class="see-p">See Project</p>
+                    </button>
```

Comment:
- [ ] Please kindly use all button effects like hover, active and pressed as mentioned in Figma design check below:

![Screenshot from 2023-02-17 17-31-04](https://user-images.githubusercontent.com/45226848/219654220-378c3af7-ffd5-4f3e-820a-ea1b145f08c8.png)